app/mailers/candidate_mailer.rb

```diff
@@ -84,6 +84,14 @@ def reference_received(reference)
     email_for_candidate(reference.application_form)
   end
 
+  def chase_candidate_decision(application_form)
+    @dbd_date = application_form.application_choices.first.decline_by_default_at.to_s(:govuk_date).strip
+    @application_choices = application_form.application_choices.select { |choice| choice if choice.offer? }
```

Comment:
This can be:

```suggestion
    @application_choices = application_form.application_choices.select(&:offer?)
```
or use the magic scope added by the enum:

```suggestion
    @application_choices = application_form.application_choices.offer
```